### PR TITLE
ISSD-2926 [Testray] Remove progress bar in Task page

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

9 files changed, 112 insertions(+), 97 deletions(-)

**Comment by rafaelfs98** [comment]
ci:test:sf

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx [hunk 1]
```diff
@@ -264,9 +270,16 @@ const Renderer: React.FC<RendererProps> = ({
...
-										)}
+										checked={
+											(Array.isArray(form[name]) &&
+												form[name]
+													.map(
+														({value}: Options) =>
+															value
+													)
+													.includes(optionValue)) ||
+											form[name].includes(optionValue)
```
Comment (JoMaAlves) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx:281]:
```suggestion
											(Array.isArray(form[name]) &&
												form[name]
													.some(
														(option: Options | string) =>
                                                            typeof option === 'string'
                                                                ? option === optionValue
                                                                : option.value === optionValue
													)
```

I believe this simplifies this statement. Could you give it a check?

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/hooks/useQueryParams.tsx [hunk 2]
```diff
@@ -76,14 +76,13 @@ const useQueryParams = () => {
...
-				serializedFilter[field.name][0]
-			);
+			const filter = SearchBuilder.in('id', serializedFilter[field.name]);
 
 			const resourceFilter = resource.includes('filter=')
 				? resource.replace(/(filter=.*?)(&|$)/, `$1 and ${filter}$2`)
-				: `${resource}&filter=${filter}`;
+				: resource.includes('?')
+				? `${resource}&filter=${filter}`
+				: `${resource}?filter=${filter}`;
```
Comment (JoMaAlves) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/hooks/useQueryParams.tsx:85]:
```suggestion
    
				?`${resource}${resource.includes('?') ? '&' : '?'}filter=${filter}`
```
Also, could you check if this makes sense?

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx [hunk 3]
```diff
@@ -238,7 +231,20 @@ const Renderer: React.FC<RendererProps> = ({
...
+						if (
+							Array.isArray(formValue) &&
+							formValue.some(
+								(option) => typeof option === 'object'
+							)
+						) {
+							formValue = formValue.map((option: any) =>
+								typeof option === 'object'
+									? option.value || option
+									: option
```
[the remark this answers is not in the document]
Comment (rafaelfs98) [reply workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx]: Thanks for the suggestion :fire:

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx (hunk 1 above)
Comment (rafaelfs98) [reply workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx]: Thanks for the suggestion :fire:

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/hooks/useQueryParams.tsx (hunk 2 above)
Comment (rafaelfs98) [reply workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/hooks/useQueryParams.tsx]: Thanks for the suggestion :fire:

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @rafaelfs98

**Comment by joseabelenda** [comment]
resent:https://github.com/brianchandotcom/liferay-portal/pull/147555

**Pull request closed without merging**